requirements.txt

```diff
@@ -1,2 +1,3 @@
 mxnet==0.10.0
 numpy>=1.12
+tensorboard>=1.0.0a3
```

Comment:
Good question.  The relevant [python API](https://github.com/dmlc/tensorboard/blob/master/python/tensorboard/writer.py) and  [proto](https://github.com/dmlc/tensorboard/blob/master/tensorboard/src/summary.proto) haven't changed since initial release.  I think we should be ok there from a compatibility point of view.  Version a3 is the first version that supports python3, any version before that will be incompatible with sockeye as it doesn't have a python3 wheel.